dotCMS/src/main/java/com/dotcms/rendering/velocity/services/ContainerLoader.java

```diff
@@ -186,6 +186,10 @@ private InputStream buildVelocity(final Container container, final String uuid,
             .append(container.getInode())
             .append("')");
 
+        velocityCodeBuilder.append("#if(!$UtilMethods.isSet($user)) ")
+        .append("#set($user = $cmsuser.getLoggedInUser($request)) ")
+        .append("#end");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "#end" appears 14 times in this file; the first occurrence is on line 108](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4504016)